.github/workflows/test.yaml

```diff
@@ -96,11 +96,16 @@ jobs:
     - name: Clone datadog-agent repository
       env:
         VERSION: ${{ github.event.inputs.datadog_agent_ref || env.DEFAULT_DATADOG_AGENT_REF }}
-      run: |
-        bash ./scripts/clone_agent.sh
-        cd $HOME/go/src/github.com/DataDog/datadog-agent
-        export VERSION_CACHE_CONTENT="${{github.event.inputs.version_cache}}"
-        if [ ! -z "$VERSION_CACHE_CONTENT" ]; then base64 -d <(echo "$VERSION_CACHE_CONTENT") > ./agent-version.cache; fi
+      uses: nick-fields/retry@v3
```

Comment:
```suggestion
      uses: nick-fields/retry@7152eba30c6575329ac0576536151aca5a72780e # v3.0.0
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Same as above